### DISCUSSION TITLE
How to add a project as card into another project board?

### DISCUSSION THREAD

**Discussion by zachguo** [community/community#21927] [Projects and Issues]
<p>I notice Github’s blog post included a screenshot of project board where each card is a project, rather than an issue or note.</p>
<p>  </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/93d5511bf1c984860b8fbb63e51194091cf74d68.png" data-download-href="/uploads/short-url/l5NdtxunQ5RJxAReHRpzap4xEnu.png?dl=1" title="34309087-16e87bbe-e705-11e7-87dd-4d521978a800" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3478203/181067568-5ed91214-c891-4928-82fc-a6caba5a9fb2.png" alt="34309087-16e87bbe-e705-11e7-87dd-4d521978a800" data-base62-sha1="l5NdtxunQ5RJxAReHRpzap4xEnu" width="690" height="354" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/93d5511bf1c984860b8fbb63e51194091cf74d68_2_690x354.png, https://aws1.discourse-cdn.com/github/optimized/1X/93d5511bf1c984860b8fbb63e51194091cf74d68_2_1035x531.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/93d5511bf1c984860b8fbb63e51194091cf74d68_2_1380x708.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/93d5511bf1c984860b8fbb63e51194091cf74d68_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">34309087-16e87bbe-e705-11e7-87dd-4d521978a800</span><span class="informations">2968×1526 396 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>How is this achieved? There seems to be no documentation about this. I tried searching </p>
<pre><code>type:project
</code></pre>
<p>or </p>
<pre><code>is:project
</code></pre>
<p>but no projects showed up.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>In <a href="https://github.com/blog/2487-track-the-progress-of-your-projects" rel="nofollow noopener">the blog post</a>, it states:</p>
<blockquote>
<p>Enter the URL of any project on GitHub into the note field to create a project summary card</p>
</blockquote>
<p>So if the project you’re trying to add a summary card for can be viewed at <a href="https://github.com/orgs/YOUR-ORGANIZATION-NAME-HERE/projects/42" rel="nofollow noopener">https://github.com/orgs/YOUR-ORGANIZATION-NAME-HERE/projects/42</a>, then you can do the following:</p>
<ol>
<li>Open the project that you want to add the card to</li>
<li>Click the + icon at the top of the column you want to add the card to</li>
<li>Paste in the URL of the project you want to add. In this hypothetical case: <a href="https://github.com/orgs/YOUR-ORGANIZATION-NAME-HERE/projects/42" rel="nofollow noopener">https://github.com/orgs/YOUR-ORGANIZATION-NAME-HERE/projects/42</a>
</li>
<li>Click the add card button</li>
</ol>
<p>I hope that helps!</p>

**Reply by pockemul** [reply]
It doesn't seem to work anymore

**Reply by bsdigi-fasa** [reply]
Any solution to it?
